build/install.sh

```diff
@@ -11,7 +11,7 @@ if [ "$(uname -s)" != "Linux" ]; then
 fi
 
 echo ":: Creating a user '$SVC_USER'..."
-useradd -m --system go-playground
+useradd -m --system go-playground --home "$DST_DIR"
```

Comment:
Good idea, but please double check that it won't break a copy operation in https://github.com/x1unix/go-playground/blob/master/build/install.sh#L17= .

If directory already exists, it might copy a directory to `/opt/go-playground/target` instead of `/opt/go-playground`.